blog/nodejs-18.md

```diff
@@ -0,0 +1,71 @@
+# Node.js 18: new fetch and test runner APIs
+
+This week Node.js v18 was released and we can found a few amazing things in this major release. Let's check the most important ones. 🚀
+
+## The fetch API
+
+Say good bye to the `node-fetch` package, now the `fetch` API is available on the global scope by default.
+
+```javascript
+const res = await fetch('https://api.belo.app/public/price');
+if (res.ok) {
+  const data = await res.json();
+  console.log(data);
+}
+```
+
+We can make requests like we do in browsers. 🙌
+
+## Test Runner module
+
+Now, we can create tests in Node.js without needing an external package.
+
+```javascript
+import test from 'node:test';
+import assert from 'node:assert';
+
+test('synchronous passing test', (t) => {
+  // This test passes because it does not throw an exception.
+  assert.strictEqual(1, 1);
+});
+```
+
+Also, we can group subtests in a parent test.
+
+```javascript
+test('top level test', async (t) => {
+  await t.test('subtest 1', (t) => {
+    assert.strictEqual(1, 1);
+  });
+
+  await t.test('subtest 2', (t) => {
+    assert.strictEqual(2, 2);
+  });
+});
+```
+
+Learn more in <https://nodejs.org/dist/latest-v18.x/docs/api/test.html>.
+
+## Prefix-only core module
+
+Did you see that strange import?
+
+```javascript
+import test from 'node:test';
+```
+
+Since this version all core modules can be imported using the `node:` prefix, there is no difference between importing `fs` and `node:fs`. But the `test` module only can be imported using the prefixed form: `node:test`.
```

Comment:
```suggestion
Since this release all core modules can be imported using the `node:` prefix, there is no difference between importing `fs` and `node:fs`. But the `test` module only can be imported using the prefixed form: `node:test`.
```